homeassistant/components/zamg/strings.json

```
@@ -0,0 +1,21 @@
+{
+  "config": {
+    "flow_title": "{name}",
+    "step": {
+      "user": {
+        "description": "Set up your ZAMG to integrate with Home Assistant.",
```

```suggestion
        "description": "Set up ZAMG to integrate with Home Assistant.",
```
It's everyone's ZAMG, isn't it? 😉